### DISCUSSION TITLE
What's the exact path of the "postal logs" / "postal logs smtp" ?

### DISCUSSION THREAD

**Discussion by XenyFord** [postalserver/postal#1607] [Help with using Postal]
Hello There,

I installed latest postal 2.0 with Caddyserver on Ubuntu 20.14, and I've been receiving lot of failure login attempts to postal smtp server as shown in the image below. So I'm in the process of enabling **fail2ban** service to prevent Bruteforce attempts to my mailserver. Can anyone please let me the exact path of ` postal logs `?

![Failed Attempts](https://user-images.githubusercontent.com/89290310/134770511-92449468-b86a-457b-b69e-ff9e41ebeb8b.jpg)

Thank you in advance!

**Comment by willpower232** [comment]
The logs are stored internally by docker but it is apparently possible to set up fail2ban, have a look at this https://www.the-lazy-dev.com/en/install-fail2ban-with-docker/
